apps/store/src/blocks/AccordionBlock.tsx

```diff
@@ -68,12 +69,15 @@ const Wrapper = styled(GridLayout.Root)({
   },
 })
 
-const Column = styled.div<{ center?: boolean }>(({ center = false }) => ({
-  gridColumn: '1 / -1',
-  [mq.lg]: {
-    gridColumn: center ? '4 / span 6' : 'span 6',
-  },
-}))
+const Column = styled.div<{ center?: boolean }>(
+  ({ center = false }) => ({
+    gridColumn: '1 / -1',
+    [mq.lg]: {
+      gridColumn: center ? '4 / span 6' : 'span 6',
+    },
```

Comment:
`confirmationPageLayoutStyles` also defines `gridColumn` for `mq.lg` breakpoint but they will only take effect when `AccordionBlock` gets rendered as a child of `ConfirmationPageBlock`? 🤔 

```suggestion
    [mq.lg]: {
      gridColumn: center ? '4 / span 6' : 'span 6',
    },
```